src/ApiConnectors/MatchesApiConnector.php

```diff
@@ -16,18 +19,20 @@ class MatchesApiConnector extends BaseApiConnector
 {
     /**
      * @param MatchSet $matchSet
+     *
      * @throws Exception
      */
```

Comment:
Missing `@return` in docblock.

Reply:
`public function send(MatchSet $matchSet): MatchSet` is the signature, no need to put it in the docblock as well. I see now that I forgot to remove the `@param`, though, so I'll be taking that as well :p